src/router/router.js

```diff
@@ -4,13 +4,15 @@ import { BrowserRouter, Route, Switch } from "react-router-dom";
 import paths from "services/shared/router-paths";
 import Dashboard from "../pages/dashboard";
 import Simulation from "../pages/simulation";
+import Login from "../pages/login";
 
 const Router = () => (
   <BrowserRouter>
     <Switch>
-      <Route path={paths.Root} exact component={Dashboard} />
-      <Route path={paths.Simulation} exact component={Simulation} />
+      <Route path={paths.Root} component={Dashboard} />
+      <Route path={paths.Simulation} component={Simulation} />
       <Route path={paths.Account} component={Account} />
+      <Route path={paths.Login} component={Login} />
```

Comment:
Exact voor de component zetten anders kun je vreemde resultaten krijgen 
```suggestion
      <Route path={paths.Root} exact component={Dashboard} />
      <Route path={paths.Simulation} exact component={Simulation} />
      <Route path={paths.Account} exact component={Account} />
      <Route path={paths.Login} exact component={Login} />
```